packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -259,6 +263,13 @@ export default class extends Component {
             ) : (
               telemetryButton
             )}
+            <TabButton
+              btnStyle='warning'
+              handler={downloadLogs}
+              handlerParam={host.uuid}
+              icon='logs'
+              labelId='hostDownloadLogs'
+            />
```

Comment:
Why not in the "Logs" tab?

Reply:
I think it might be confusing because they are not the same at all, the first ones are messages from XAPI and the other ones are system logs from `/var/logs` on the host.

What do you think?

Reply:
I think it's confusing to you because you know exactly what the Logs tab contains but for someone who's looking for logs, no matter what kind, it would make sense to naturally go to the Logs tab.